shared/utils/yaml-to-shorthand.py

```diff
@@ -21,6 +21,14 @@
 from ssgcommon import open_yaml, required_yaml_key
 
 
+def add_warnings(element, warnings):
```

Comment:
IMO rename to add_warning_elements and move after add_sub_element. You are using add_sub_element in the function so it helps when it's defined after you define add_sub_element